code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -674,9 +674,8 @@ emp_act
 			switch(M.damtype)
 				if("brute")
 					if(M.force > 35) // durand and other heavy mechas
-						Paralyse(1)
-					else if(M.force > 20 && !IsWeakened()) // lightweight mechas like gygax
-						Weaken(2)
+						drop_item()
```

Comment:
Might want to give some feedback to the user saying they dropped it or something. It's easy to miss especially since they don't get pushed down anymore